engine/include/cubos/engine/scene/bridge.hpp

```diff
@@ -25,35 +25,34 @@ namespace cubos::engine
     ///     },
     ///     "entities": {
     ///         "baz": {
-    ///             "cubos/position": {
-    ///                 "x": 1,
-    ///                 "y": 2,
-    ///                 "z": 3
+    ///             "cubos::engine::Position": {
+    ///                 "x": 10
     ///             }
     ///         },
     ///         "foo.bar": {
-    ///             "cubos/parent": "baz",
+    ///             "Parent": "baz",
     ///         }
     ///     }
     /// }
     /// @endcode
     ///
-    /// This scene will import the subscene with ID `6f42ae5a-59d1-5df3-8720-83b8df6dd536`, and
+    /// This scene will import the sub-scene with ID `6f42ae5a-59d1-5df3-8720-83b8df6dd536`, and
     /// prefix all of its entities with `foo.`. The entity `foo.bar` will override the entity `bar`
     /// from the imported scene, while the entity `baz` will be added to the scene.
     ///
     /// @ingroup scene-plugin
-    class SceneBridge : public AssetBridge
+    class SceneBridge : public FileBridge
     {
     public:
         /// @brief Constructs a bridge.
         ///
         SceneBridge()
-            : AssetBridge(typeid(Scene))
+            : FileBridge(typeid(Scene))
```

Comment:
`FileBridge` from #795 kinda changed how things work here, maybe to avoid conflicts, rebase and use the new version?

Reply:
Wait and #795 depends on this PR :V

Reply:
Feel free to merge the other one, it uses just a single commits from this PR, its okay :relieved: I'll rebase it on main later